.github/workflows/unused.yml

```diff
@@ -6,14 +6,14 @@ on:
       - develop
     paths:
       - '**.php'
-      - 'composer.**'
```

Comment:
I believe these need to stay as they are currently: 

https://docs.github.com/en/github-ae@latest/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

Reply:
Why? What's wrong?

```
*: Matches zero or more characters, but does not match the / character. For example, Octo* matches Octocat.
```

Reply:
Maybe it doesn't need to match `/` for our needs. I believe @MGatner wrote this, maybe he can chime in.

Reply:
The code comes from the latest devkit:
https://github.com/codeigniter4/devkit/blob/develop/src/Template/.github/workflows/unused.yml

Reply:
Yes this was a mistake, it should be a single asterisk. Thanks for checking on my work though 🤗